config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Service.java

```diff
@@ -95,6 +95,10 @@ public String getMappingValue() {
     @Override
     public void preWrite(File configFolder, DocumentFileUtils documentFileUtils) {
         setPolicy(getPath());
+
```

Comment:
add some comment here why this is required

Reply:
Removed code